app/views/embassies/index.html.erb

```diff
@@ -1,11 +1,9 @@
 <% page_title "Find a British embassy, High Commission or Consulate" %>
 <% page_class "worldwide-organisations embassies-index govuk-width-container" %>
-<header class="block headings-block">
```

Comment:
removing this makes sense, i'm seeing these extra headers in a few places.
https://github.com/alphagov/frontend/pull/2775

Reply:
I reckon it was just how govuk used to be structured and was forgotten about in Whitehall. As we discussed on a previous PR, we already have a header in our default template so removing this one makes sense.